### ISSUE TITLE
FLARE

### ISSUE THREAD

**Issue by andrearosasco** [andrearosasco/papers#24]
#### Architecture
It takes a simple Diffusion Transformer trained with flow-matching loss and add a new objective. In a way similar to "Transfusion" the transformer also takes some future prediction token that are used to predict an embedding to the next observation. Basically generating the action while forcing the architecture to "think ahead" to the next state improves performance. What's more it can be pre-trained on action-less video just by optimizing the next-observation loss and not the action-loss.

The ground truth is an embedding generated by a pre-trained VL model. The model is pre-trained on OpenX end-to-end with flow-matching so that its embedding are action-aware.

#### Takeaways
While LAPA is more of a way to introduce actionless videos at pretraining time, FLARE does that at fine-tuning. They show that they can use as little as 1 teleoperated demonstration, add human actionless video of the same task and get much higher performance.

#### Benchmarks
Sim: Robocasa (single arm), Simulated GR1 (from gr00t)
Real: real GR1
